src/main/java/bio/terra/service/dataset/DatasetTableDao.java

```diff
@@ -28,30 +30,30 @@
     private static final Logger logger = LoggerFactory.getLogger(DatasetTableDao.class);
 
     private static final String sqlInsertTable = "INSERT INTO dataset_table " +
-        "(name, raw_table_name, soft_delete_table_name, dataset_id, primary_key, partition_mode, " +
-        "partition_column, int_partition_min_value, int_partition_max_value, int_partition_interval) " +
-        "VALUES (:name, :raw_table_name, :soft_delete_table_name, :dataset_id, :primary_key, :partition_mode, " +
-        ":partition_column, :int_partition_min_value, :int_partition_max_value, :int_partition_interval)";
+        "(name, raw_table_name, soft_delete_table_name, dataset_id, primary_key, bigquery_partition_config) " +
+        "VALUES (:name, :raw_table_name, :soft_delete_table_name, :dataset_id, :primary_key, " +
+        "cast(:bigquery_partition_config AS json))";
     private static final String sqlInsertColumn = "INSERT INTO dataset_column " +
         "(table_id, name, type, array_of) VALUES (:table_id, :name, :type, :array_of)";
     private static final String sqlSelectTable =
-        "SELECT id, name, raw_table_name, soft_delete_table_name, primary_key, partition_mode, " +
-        "partition_column, int_partition_min_value, int_partition_max_value, int_partition_interval " +
+        "SELECT id, name, raw_table_name, soft_delete_table_name, primary_key, bigquery_partition_config::text " +
         "FROM dataset_table WHERE dataset_id = :dataset_id";
     private static final String sqlSelectColumn = "SELECT id, name, type, array_of FROM dataset_column " +
         "WHERE table_id = :table_id";
 
     private final DataSource jdbcDataSource;
     private final NamedParameterJdbcTemplate jdbcTemplate;
 
+    @Autowired private ObjectMapper objectMapper;
```

Comment:
The norm for our code is to list all `@Autowired` in the constructor and assign. That is what is happening right below this autowire.